identity-and-access-management/secret-manager/quickstart.mdx

```diff
@@ -20,93 +20,42 @@ Scaleway’s [Secret Manager](/identity-and-access-management/secret-manager/con
-4. Click <Icon name="plus" /> and **Create version**. A pop-up displays.
-5. Add your version manually or click <Icon name="plus" /> to import it from a file.
-6. Click <Icon name="toggle" /> to enable or disable the version.
-7. Click **Add new version**.
+6. Click **Create secret**. The **Overview** tab of your secret displays. You can copy your secret's ID, add a description, tags or delete your secret.
+7. Click **Back to Secrets** to display your folder and its contents.
 
-## How to manage a version of a secret
-
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets display.
-2. Click the secret you want to add a version to. Your secret's overview page displays.
-3. Click the **Versions** tab. The versions of your secret display.
-4. Click <Icon name="more" /> next to the version you want to manage.
-5. Click **Disable** to disable one specific version of your secret.
-6. Click **Copy version's value** to copy the value of your version.
-
-## How to delete your secret
-
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets display.
-2. Click <Icon name="more" /> next the secret you want to delete.
-3. Click **Delete**. A pop-up displays.
-4. Type **DELETE** and click **Delete secret**.
-
-<Message type="important">
-  Deleting a secret is a permanent action. It erases every version you have created for your secret.
-</Message>
-
-## How to delete a version of a secret
-
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets display.
-2. Click the secret for which you want to delete the version. Your secret's overview page displays.
-3. Click the **Versions** tab.
-4. Click <Icon name="more" /> next to the version you want to delete.
-5. Click **Delete**. A pop-up displays.
-6. Type **DELETE** and click **Delete version**.
-
-<Message type="important">
-  Deleting a version is a permanent action. You will not be able to use the version again.
-</Message>
-
-## How to delete a folder
+    <Message type="tip">
+      - Read our [documentation](/identity-and-access-management/secret-manager/how-to/create-folder/) to learn how to create more folders once you have created resources in Secret Manager.
+      - Read our [documentation](/identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders/) to find out how to migrate secrets located in the `/` path, to specific folders.
+    </Message>
 
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets and folders display.
-2. Click the <Icon name="delete" /> icon next to the folder you want to delete.
-3. Type **DELETE** to confirm and click **Delete folder**.
+## How to create a secret version
```

Comment:
Maybe we can say that at its creation, a secret has one version, and that the user can add MORE